src/main/java/lesson_3/Task_21.java

```diff
@@ -0,0 +1,47 @@
+/*
+Напишите три цикла выполняющих многократное сложение строк, один с
+помощью оператора сложения и String, другой с помощью StringBuilder и метода
+append, а также аналогино для StringBuffer. Сравните скорость их выполнения.
+ */
+public class Task_21 {
+    public void showTask_21(String text) {
+        System.out.println("1.Слаживаем с помощю цикла For : ");
+        showAddString(text);
+        System.out.println();
+        System.out.println("2.Слаживаем с помощю StringBuilder метод append : ");
+        showBuilderAppend(text);
+        System.out.println();
+        System.out.println("3.Слаживаем с помощю StringBuffer  метод append : ");
+        showBufferAppend(text);
+
+    }
+
+    public void showAddString(String text) {
+        long time = System.nanoTime();
+        for (int i = 1; i <= 2; i++) {
+            text += text;
+        }
+        System.out.println("Обедененый текст : " + text);
+        time = System.nanoTime() - time;
+        System.out.printf("Метот выполнен за %,9.3f ms\n", time / 1_000_000.0);
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.